etl/steps/data/meadow/un_sdg/2022-05-26/un_sdg.py

```diff
@@ -0,0 +1,404 @@
+import requests
+import numpy as np
+import re
+import itertools
+import math
+import pandas as pd
+from structlog import get_logger
+from pathlib import Path
+from typing import Tuple
+
+from owid.walden import Catalog
+from owid.catalog import Dataset, Table, DatasetMeta, TableMeta
+from owid.catalog.utils import underscore
+
+BASE_URL = "https://unstats.un.org/sdgapi"
+log = get_logger()
+
+
+def run(dest_dir: str, query: str = "") -> None:
+    # retrieves raw data from walden
+    version = Path(__file__).parent.stem
+    fname = Path(__file__).stem
+    namespace = Path(__file__).parent.parent.stem
+
+    version = "2022-05-26"
```

Comment:
`version` is already defined above (both string and `Path(__file__).parent.stem` are good solutions).